client/src/App.js

```diff
@@ -0,0 +1,141 @@
+import { useCallback, useEffect, useState } from 'react';
+
+const App = () => {
+  const [text, setText] = useState('');
+
+  const handleChange = (e) => {
+    e.preventDefault();
+
+    setText(e.currentTarget.value);
+  };
+
+  const [result, setResult] = useState(undefined);
+
+  const handleReset = () => {
+    setText('');
+    setResult(undefined);
+  };
+
+  const handleFetch = (e) => {
+    e.preventDefault();
+
+    fetch('http://0.0.0.0:5000/api/classify', {
```

Comment:
Please add "configuration variable" with `http://0.0.0.0:5000/api/` and reuse to make a path here.
Not sure what is proper react way - global `const` somewhere at the top of the file is fine for me.

Reply:
Yes, it will be added later, probably with next PR.